app/graphql/mutations/save_account_mutation.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Mutations
+  # GraphQL mutation to update the current user account
+  class SaveAccountMutation
+    def call(user, args, _ctx)
```

Comment:
`set_minimum_password_length` sets a parameter for the views. It's not important for an API-only backend.